Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+# TODO: Delete when available at rubygems
+gem 'chef-dk', github: 'chef/chef-dk',
+               branch: 'ee88c69cd1b6f5e47ae321b4d5af645340bf7f2d'
```

Comment:
I'm not going lie...this worries me. Bringing ChefDK (and thus Chef) along with all of its deps in seems dangerous. I would think we would want to make make the dependency tree as small as possible for this framework.